CHANGELOG.md

```diff
@@ -1,5 +1,51 @@
 # Changelog
 
+## 0.0.6
+
+Released on 2025-12-23.
+
+### Bug fixes
+
+- Avoid expanding type aliases in implicit tuple aliases ([#22015](https://github.com/astral-sh/ruff/pull/22015))
+- Distribute `type[]` over unions ([#22115](https://github.com/astral-sh/ruff/pull/22115))
+- Exclude parameterized tuple types from narrowing when disjoint from comparison values ([#22129](https://github.com/astral-sh/ruff/pull/22129))
+- Respect debug text interpolation in f-strings ([#22151](https://github.com/astral-sh/ruff/pull/22151))
+
+### LSP server
+
+- Add `ty.configuration` and `ty.configurationFile` options ([#22053](https://github.com/astral-sh/ruff/pull/22053))
+- Add `diagnosticMode: off` to disable diagnostics while retaining Go To Definition, etc. ([#22073](https://github.com/astral-sh/ruff/pull/22073))
+- Set flag to avoid `type[T@f]` being inserted when you double-click on the inlay ([#22139](https://github.com/astral-sh/ruff/pull/22139))
+- Use Markdown for completions documentation ([#21752](https://github.com/astral-sh/ruff/pull/21752))
+
+### CLI
+
+- Abort printing diagnostics when pressing Ctrl+C ([#22083](https://github.com/astral-sh/ruff/pull/22083))
+
+### Configuration
+
+- Add `respect-type-ignore-comments` configuration option ([#22137](https://github.com/astral-sh/ruff/pull/22137))
+
+### Other changes
+
+- Bind self with instance in `__get__` ([#22155](https://github.com/astral-sh/ruff/pull/22155))
+- Support custom builtins ([#22021](https://github.com/astral-sh/ruff/pull/22021))
+- Support type inference between protocol instances ([#22120](https://github.com/astral-sh/ruff/pull/22120))
+- Synthesize a `_fields` attribute for NamedTuples ([#22163](https://github.com/astral-sh/ruff/pull/22163))
+- Synthesize a `_replace` method for NamedTuples ([#22153](https://github.com/astral-sh/ruff/pull/22153))
+- Narrow tagged unions of `TypedDict` ([#22104](https://github.com/astral-sh/ruff/pull/22104))
```

Comment:
These should all go in like... a "Typing" section.

Reply:
Sounds good to me -- let's just rename this section?